lib/hexo/load_plugins.js

```diff
@@ -12,25 +12,38 @@ module.exports = ctx => {
 };
 
 function loadModuleList(ctx) {
-  if (ctx.config && Array.isArray(ctx.config.plugins)) {
-    return Promise.resolve(ctx.config.plugins).filter(item => typeof item === 'string');
+  let customThemeName;
+
+  if (ctx.config) {
+    const { theme, plugins } = ctx.config;
+
+    if (Array.isArray(plugins)) {
+      return Promise.resolve(ctx.config.plugins).filter(item => typeof item === 'string');
```

Comment:
`ctx.config.plugins` -> `plugins`

Reply:
I am not sure whether the context is required when return. I will have a test before changing it.